docs/index.rst

```diff
@@ -9,7 +9,7 @@ LibMultiLabel is a library for binary, multi-class, and multi-label classificati
 
 LibMultiLabel can be used through either `Command Line Interface <cli_index.html>`_ or `APIs <library_index.html>`_. Check their respective documents for installation instructions.
 
-For practical tips, please see the `User Guide <https://www.csie.ntu.edu.tw/~cjlin/papers/libmultilabel/userguide.pdf>`_.
+For practical use, please see the `Tutorial <https://www.csie.ntu.edu.tw/~cjlin/libmultilabel/tutorial.html>`_.
```

Comment:
Tutorial -> Tutorials

can we have a relative address?? For user guide we need a hard link because it's not part of the code repository